### ISSUE TITLE
crnn训练代码

### ISSUE THREAD

**Issue by caoyangcr7** [chineseocr/chineseocr#262]
感谢作者的贡献，在此提出一个小问题，在crnn的训练代码中，训练的时候代码中并未定义transformer(也就是读进来的图片会是0-255的矩阵)，但是在预测代码函数predict中进行了resizeNormalize操作，读进去的图片是-1~1的图片矩阵。

**Comment by wenlihaoyu** [comment]
请查看train/ocr/train-ocr.ipynb文件   
train_loader = torch.utils.data.DataLoader(
    traindataset, batch_size=batchSize,
    shuffle=False, sampler=None,
    num_workers=int(workers),
    collate_fn=alignCollate(imgH=imgH, imgW=imgW, keep_ratio=keep_ratio))

**Comment by caoyangcr7** [comment]
谢谢回复。train-ocr.ipynb中pathdataset transform默认为None，且并未传入参数。

**Comment by wenlihaoyu** [comment]
alignCollate函数处理了的，

**Comment by caoyangcr7** [comment]
😵😵😵确实是的。谢谢回复

**Comment by zlszhonglongshen** [comment]
> 感谢作者的贡献，在此提出一个小问题，在crnn的训练代码中，训练的时候代码中并未定义transformer(也就是读进来的图片会是0-255的矩阵)，但是在预测代码函数predict中进行了resizeNormalize操作，读进去的图片是-1~1的图片矩阵。

你好问下，能否分享一下，crnn的训练代码呢？谢谢 593956670@qq.com

**Comment by caoyangcr7** [comment]
@zlszhonglongshen master branch里面有。

**Comment by zlszhonglongshen** [comment]
> @zlszhonglongshen master branch里面有。

找到了 谢谢
